README.md

```diff
@@ -8,9 +8,42 @@ Please report bugs on [phabricator](https://phabricator.wikimedia.org/project/vi
 
 _**Prerequisites: This repository requires node v16 and up**_
 
+### One-time setup
```

Comment:
"One-time setup" sounds a bit strange to me. How about "Initial setup"?

Reply:
Sure, done.